src/convertToObject.js

```diff
@@ -1,12 +1,19 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString
+    .split(';')
+    .filter(line => line.trim())
+    .forEach(line => {
+      const [property, value] = line.split(':');
+      if (property && value) {
+        const key = property.trim();
+        const val = value.trim();
```

Comment:
[CODE STYLE] - The variable names `property` and `value` are good, but `key` and `val` could be more descriptive. Consider renaming `key` to `trimmedProperty` and `val` to `trimmedValue` to increase code readability.